src/app/designationEditor/websiteModal/websiteModal.tpl.html

```diff
@@ -1,8 +1,10 @@
 <div class="modal-body">
   <div class="form-group">
-    <label>Website Address</label>
-    <input name="website" type="text" class="form-control" placeholder="http://" ng-model="$ctrl.website">
-    <span class="help-block">Fill in this field to add a website link.</span>
+    <label>
+      Website Address
+      <input name="website" type="text" class="form-control" placeholder="https://" aria-describedby="website-help" ng-model="$ctrl.website">
```

Comment:
Unrelated to this PR, but someday we might want to add the protocol to the address they put in here if it is missing, it is a common source of mistakes on these pages.